airflow/cli/cli_parser.py

```diff
@@ -540,9 +540,18 @@ def string_lower_type(val):
 )
 ARG_IGNORE_DEPENDS_ON_PAST = Arg(
     ("-I", "--ignore-depends-on-past"),
-    help="Ignore depends_on_past dependencies (but respect upstream dependencies)",
+    help="Deprecated -- use `--depends-on-past ignore` instead. "
+    "Ignore depends_on_past dependencies (but respect upstream dependencies)",
     action="store_true",
 )
```

Comment:
I just realised we could improve this further by doing `dest="depends_on_past", action="store_const", const="check"` here and merge the two arguments. We can do this here or in a subsequent PR.

We should probably also make the two arguments mutually exclusive (`add_mutually_exclusive_group`), although I’m not quite sure how that can be done with how Airflow manages arguments.